src/api/worker/facades/lazy/CalendarFacade.ts

```diff
@@ -456,7 +454,8 @@ export class CalendarFacade {
 		const allAlarms = flat(
 			userAlarmInfosAndNotificationsPerEvent.map(({ userAlarmInfoAndNotification }) => userAlarmInfoAndNotification.map(({ alarm }) => alarm)),
 		)
-		const alarmIds = await this.entityClient.setupMultipleEntities(userAlarmInfoListId, allAlarms)
+
+		const alarmIds: Array<string> = await this.entityClient.setupMultipleEntities(userAlarmInfoListId, allAlarms)
```

Comment:
`Id[]`

Reply:
`Array<Id>`, fite me

Reply:
I thought they wanted to deprecate this syntax but I'm not sure anymore